scripts/configure_valhalla.sh

```diff
@@ -116,6 +116,32 @@ files=$(echo $files | xargs)
 
 # be careful how to write the config (mostly for restart scenarios where env vars are true all of a sudden)
 if test -f "${CONFIG_FILE}"; then
+
+  if [[ "${update_existing_config}" == "True" ]]; then 
+    echo "INFO: Found existing valhalla.json. Updating possibly missing entries."
+
+    # create temporary default config
+    valhalla_build_config > ${TMP_CONFIG_FILE}  || exit 1
+
+    # for each path in the temp config (excluding array indices)
+    jq -r 'paths | select(map(type) | index("number") | not ) | "." + join(".")' ${TMP_CONFIG_FILE} | while read key ; do
```

Comment:
`paths` selects path to all leaf nodes, including array items. The output of this looks like this:

```
.meili
.meili.default
.meili.default.beta
...
```

Reply:
well it's pretty much chinese to me anyways and I'm happy I didn't have to give this a shot:)